src/Raven.Server/Documents/Queries/Results/IQueryResultRetriever.cs

```diff
@@ -7,14 +7,18 @@
 using Lucene.Net.Store;
 using Raven.Server.Documents.Indexes;
 using IndexSearcher = Lucene.Net.Search.IndexSearcher;
+using Sparrow;
+using Sparrow.Server;
 
 namespace Raven.Server.Documents.Queries.Results
 {
     public interface IQueryResultRetriever
     {
         (Document Document, List<Document> List) Get(ref RetrieverInput retrieverInput, CancellationToken token);
 
-        bool TryGetKey(ref RetrieverInput retrieverInput, out string key);
+        bool TryGetKeyLucene(ref RetrieverInput retrieverInput, out string key);
+
+        bool TryGetKeyLucene(Corax.IndexSearcher searcher, long id, out UnmanagedSpan key);
```

Comment:
`TryGetKeyCorax()`?

Reply:
Sorry for the inconvenience. I accidentally changed all overloads instead of just this one. Fixed :)